modules/acknowledgements/php/NDB_Menu_Filter_Form_acknowledgements.class.inc

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * This class provides the code to create the menu filter form that allows
+ * for data entry of entities that should be acknowledged in publications
+ *
+ * PHP Version 5
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   Justin Kat <justin.kat@mail.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once 'NDB_Menu_Filter_Form.class.inc';
```

Comment:
This should get autoloaded by composer's autoload file.. including it explicitly breaks some things such as automated test coverage reporting.